All EIPIP Meetings/Meeting 060.md

```diff
@@ -0,0 +1,121 @@
+
+**Micah:** [troll](https://www.youtube.com/watch?v=OlKWquEmJms&t=3360) will come show up and sue the person for using their patented stuff it's the most ridiculous thing ever and the fact that this is an issue in the world makes me want to fast forward my plans to murder all of humanity yeah so anyways like i just think we should allow other licenses in cc0 i'm totally fine with like the default being cc0 only and if somebody wants something else like somebody has i mean somebody has to prove it anyways but yeah can we can we have like cc people have like in in wikipedia the content licenses cc by shop and you can license it with other license so you people can cite your content with either one can we make cc buy as one required license and and then if they want to license it in any other ways they can declare that we also license it under other forms so if you know you if you worry about patents we happen to also license it with this this license so you can use it how's that i would be okay with that i don't think that addresses the problem that people have i think the problem people have is they want to include in their assets directory code that is already code that someone else wrote that they are copying as part of their eip usually it's open zeppelin so opens up i think license everything they do is mit um and so they want to copy that mit code drop it in the assets directory and then use it in their reference implementation but they can't re-license that as cc0 because they don't have the rights to do so this specific case i think it's impossible to avoid it that right people can always propose something and then put it there even though it can be put assumingly uh influencing other people's patent who don't necessarily are the authors right people can definitely lie and we just assume that everybody's telling the truth in this case because that gives us legal protection i think is the general idea like if uh if someone steals someone else's code what's in the github repository then a patent troll comes around generally the person who put it there is the person who's in trouble and if as long as they like if they claimed that this is non-patented but it was that's not true then they're the ones that are in trouble and so the ultimately like we just need them from a legal standpoint presumably we need them to make a declaration that this is x and that's enough whether that declaration is true or not doesn't actually matter we just need them to you know effectively claim make a claim somewhere and yeah i wonder if that is that that is achieved by a license or another the the the thing i'm concerned about is having people allowing people to use some license that's actually restricted of distribution of the content um yeah and that's why i prefer cc0 only um like i want everything like i want to say someone goes and reads an eip and that includes all aspects of the aap including the 
```

Comment:
```suggestion
**Micah:**  and then the patent [troll](https://www.youtube.com/watch?v=OlKWquEmJms&t=3360) will come show up and sue the person for using their patented stuff. it's the most ridiculous thing ever and the fact that this is an issue in the world makes me want to fast forward my plans to murder all of humanity 

**Sam**: yeah so anyways like i just think we should allow other licenses in cc0 i'm totally fine with like the default being cc0 only and if somebody wants something else like somebody has i mean somebody has to prove it anyways but yeah 

**Victor**: can we have like people have like in in wikipedia the content licenses cc by shop and you can license it with other license so you people can cite your content with either one can we make cc buy as one required license and and then if they want to license it in any other ways they can declare that we also license it under other forms so if you know you if you worry about patents we happen to also license it with this this license so you can use it how's that

 **Micah**:  i would be okay with that i don't think that addresses the problem that people have i think the problem people have is they want to include in their assets directory code that is already code that someone else wrote that they are copying as part of their eip usually it's open zeppelin so opens up i think license everything they do is mit um and so they want to copy that mit code drop it in the assets directory and then use it in their reference implementation but they can't re-license that as cc0 because they don't have the rights to do
 
  **Victor**:  so this specific case i think it's impossible to avoid it that right people can always propose something and then put it there even though it can be put assumingly uh influencing other people's patent who don't necessarily are the authors right 
  
  **Micah**: people can definitely lie and we just assume that everybody's telling the truth in this case because that gives us legal protection i think is the general idea like if uh if someone steals someone else's code what's in the github repository then a patent troll comes around generally the person who put it there is the person who's in trouble and if as long as they like if they claimed that this is non-patented but it was that's not true then they're the ones that are in trouble and so the ultimately like we just need them from a legal standpoint presumably we need them to make a declaration that this is x and that's enough whether that declaration is true or not doesn't actually matter we just need them to you know effectively claim make a claim somewhere and 
  
 **Victor**: yeah i wonder if that that is achieved by a license or another the thing i'm concerned about is having people allowing people to use some license that's actually restricted of distribution of the content um 
 
 **Micah**: yeah and that's why i prefer cc0 only um like i want everything like i want to say someone goes and reads an eip and that includes all aspects of the eip including the 
```